docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -127,9 +127,49 @@ BigDL-LLM for GPU supports on Linux has been verified on:
 
       * Step 2: Download and install `Intel® oneAPI Base Toolkit <https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html>`_ with version 2024.0. OneDNN, OneMKL and DPC++ compiler are needed, others are optional.
 
-        .. seealso::
+      Intel® oneAPI Base Toolkit 2024.0 install methods:
+
+      .. tabs::
+         .. tab:: Offline installer
```

Comment:
put this at the very end since it is not recommended.
And can we collapse this?

Reply:
Sure. However, command line installer allows user to pick install location. So we can use it to install multiple versions of oneAPI if required. We cannot do that with apt-get installer.

Reply:
I understand. The reason may be the offline installer may not be available always for users, thus apt-get is more recommended.